examples/schema/permissioned/book.graphql

```diff
@@ -0,0 +1,5 @@
+type Book @policy(id:"7dc51aabc0248cf106265c902bf56faa1989ec41a6bbd36b6e438cfade7aee4a", resource:"book") {
```

Comment:
Document how? like with a string? I use the example as arguments to `defradb cli command` with `-f` options and don't want invalid comments to cause syntax error.

Reply:
> Document how? like with a string?

Yes, if `// fdjnsgka` doesnt work, try `# gfsdh`.  I care quite a lot less about Source dev convience than what users will see when they look inside the top-level `examples` directory from the repo root too - if I had to choose one to break I'd rather spoil our ability to use them as you described than to have an unreadable set of examples for our users.

If you really cannot find a means to document them, please move them to a slightly more obscure location.

Reply:
What specifically would you like documented? It's a `Book` schema with policy specified on it?

You are happy with the removal if I don't find time to document this in this PR? rather than to keep like this?

I do very much disagree BTW.

Reply:
> What specifically would you like documented? It's a Book schema with policy specified on it?

```
/*
The below sdl contains an example `Book` object with an example source hub policy id
and resource name.

The policy id must exist in sourcehub first, the resource name must exist on the policy. The resource name
does not need to be similar to the collection name.

etc etc (normal user-friendly documentation that you'd expect in an `/examples` directory)
*/
```

Reply:
- [ ] Add suggested documentation to public examples